js/index.js

```diff
@@ -285,16 +280,45 @@ $(function() {
         $('#showjs').show();
       }
     });
+    var publishNewGist = function() {
+        if (!confirm('Do you really want to publish as a Gist?')) { return; }
```

Comment:
Maybe add something here along the lines of "This code will be publicly visible"? And add the word "anonymous" before Gist so it's clear it's not associated to the user's account if they happen to be logged in too?